dev-lang/go/go-9999.ebuild

```diff
@@ -169,6 +186,15 @@ src_install()
 	insinto /usr/lib/go/pkg/tool
 	doins -r "pkg/tool/$(go_tuple)"
 	fperms -R +x /usr/lib/go/pkg/tool
+
+	if use cros_host && use arm64-extras; then
+		insinto /usr/lib/go/pkg
+		doins -r pkg/linux_arm64
+
+		insinto /usr/lib/go/pkg/tool
+		doins -r pkg/tool/linux_arm64
```

Comment:
It my not be needed  I don't remember if it was (I made this patch a long time ago).  I'll check it.